__tests__/endpoints.test.js

```diff
@@ -0,0 +1,11 @@
+const request = require("supertest");
+const app = require("../app/app");
+const endpoints = require("../endpoints.json");
+
+describe("/api", () => {
+  test("GET /api - success", async () => {
```

Comment:
Test description?

Reply:
Fair point I was rushing a bit and could've been more descriptive 😬 I've refactored all the descriptions to be a bit more useful (hopefully)